README.md

```diff
@@ -24,6 +24,11 @@ optimized versions of the same program.
 
 ### Phase 0
 
+* [Formal Verification of the Eth2.0 Specs in Dafny](https://github.com/ConsenSys/eth2.0-dafny) The objective of this project is to write a formal specification of the Eth2.0 specification in the verification-aware programming language [Dafny](https://github.com/dafny-lang/dafny/wiki).
```

Comment:
Can you also please move both items after the Deposit Contract items? I'm trying to keep it sorted alphabetically.